src/deluge/model/mod_controllable/mod_controllable_audio.cpp

```diff
@@ -616,31 +616,37 @@ void ModControllableAudio::processSRRAndBitcrushing(StereoSample* buffer, int32_
                                                     ParamManager* paramManager) {
 	StereoSample const* const bufferEnd = buffer + numSamples;
 
-	uint32_t bitCrushMaskForSRR = 0xFFFFFFFF;
+	uint32_t bitCrushMask = 0xFFFFFFFF;
 
 	bool srrEnabled = isSRREnabled(paramManager);
 
 	// Bitcrushing ------------------------------------------------------------------------------
 	if (isBitcrushingEnabled(paramManager)) {
-		uint32_t positivePreset =
-		    (paramManager->getUnpatchedParamSet()->getValue(params::UNPATCHED_BITCRUSHING) + 2147483648) >> 29;
-		if (positivePreset > 4) {
-			*postFXVolume >>= (positivePreset - 4);
+		int32_t preset =  paramManager->getUnpatchedParamSet()->getValue(params::UNPATCHED_BITCRUSHING);
+		// Scale to 1-50, clamp to 15 -- use that as the reduction. Ie. reduce 1 bit per number
+		// on the user scale. Top end above 15 is all 1 bit sound.
+		//
+		// TODO: This changes sound of existing instruments with bitcrush. Need to address
+		// in load / save code.
+		int32_t crushAmount = std::min<int32_t>(15, preset / 85899345 + 25);
+		// TODO: What's the logic in the volume compensation here? Is this ad-hoc, or something
+		// that makes specific sense?
```

Comment:
seems ad hoc and I don't know that it matches the bitcrush volume increase all that well